st2common/st2common/router.py

```diff
@@ -178,6 +178,9 @@ def add_spec(self, spec, transforms):
                     for transform in transforms[filter]:
                         m.connect(None, re.sub(filter, transform, path), **connect_kw)
 
+                    module_name = endpoint['operationId'].split(':', 1)[0]
```

Comment:
You talking about overhead comparing to state before or after 2727? We've imported them during init before the spec thing to we should be back to the same level we were before the PR.